libs/designsystem/src/lib/components/page/page.component.scss

```
@@ -23,6 +23,17 @@ ion-toolbar {
   --padding-top: 0;
   --padding-bottom: 0;
 
+  box-sizing: border-box;
+  padding-left: #{utils.size('xs')};
+  padding-right: #{utils.size('xs')};
+  transition: padding utils.get-transition-easing('enter-exit')
+    utils.get-transition-duration('short');
+
+  &.content-scrolled {
+    padding-left: 0;
+    padding-right: 0;
```

Suggestion: Use logical property

```suggestion
    padding-inline: 0;
```